---
layout: home
title: "Post Template using Rmd"
date: 2015-01-01
title: "Accessions"
author: "Jinliang Yang"
categories: [tutorial, general]  
tags: [github, general, markdown, jekyll, rstats, servr]  
comments: true  
archive: true

---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
library("googleVis")
op <- options(gvis.plot.tag='chart')
```

## GBS Data Summary:

```{r results='asis', echo=FALSE, warning=FALSE, error=FALSE, message=FALSE}
#library(data.table)
df <- read.csv("~/Documents/Github/zbd/data/gbs_by_pop.csv")

Table <- gvisTable(df, formats=list(Size="#,###"))
plot(Table)

```



## GRIN Germplasm 

Total Number of Accessions in GRIN genebank: [N=13,131](../page/accession/index.html)

```{r results='asis', echo=FALSE, warning=FALSE, error=FALSE, message=FALSE}
#library(data.table)
d <- read.csv("~/Documents/Github/zbd/data/grin_by_country.csv")
#names(d)

library("googleVis")
Geo=gvisGeoChart(d, locationvar="c1",  colorvar="Freq",
                 options=list(projection="kavrayskiy-vii"))
plot(Geo)

```

## Data summary

```{r results='asis', echo=FALSE, warning=FALSE, error=FALSE, message=FALSE}
df=data.frame(country=c("WGS", "GBS", "SNP-Chip", "RNA-seq", "ChIP-seq", "MNase"), 
              Geno=c(10,13,14, 10,13,14),
              Size=c(10,13,14, 10,13,14)*2,
              Study=c(1,2,3,1,2,3))

Column <- gvisColumnChart(df)
plot(Column)
```

## Mapping Populations



