---
layout: post
title: "Atlas and Time Series RNA-seq Data Sets"
date: 2016-11-18
author: Jinliang Yang

categories: [RNA-seq]  
tags: [RNA-seq, atlas, maize, NGS]  
comments: true  
archive: false
---




Some RNA-seq data sets for certain lines collected on different tissues or measured at different developmental stages.


------

### 1. RNA-Seq with diverse B73 tissues [Stelpflug et al., 2015](https://dl.sciencesocieties.org/publications/tpg/articles/9/1/plantgenome2015.04.0025).

FPKM matrices are available from the above paper.

- **Study description**: Microarray versus RNA-Seq comparative analysis with diverse B73 tissues.
- **SRA Study ID**: [SRP010680](https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP010680)
- **Data Size**: 50 Gbp; and 31Gb in size



### 2. Dry seeds (0 hour) to hour 72 every six hours from [Liu et al., 2013](http://www.pnas.org/content/110/10/3979.full).

- **Study description**: To discover the transcriptional dynamics during seed germination we 
have obtained the time course transcriptomes for embryonic shoot apical meristem (SAM) every six hours, starting from dry seeds to hour 72 (3 days).
- **SRA Study ID**: [SRP017122](https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP017122)
- **Data Size**: 1.4 Tbp; and 0.9Tb in size
- Zea mays subsp. mays strain: White Crystal


### 3. Dry seeds (0 hour)) to hour 192 from [Yu et al., 2015](https://www.ncbi.nlm.nih.gov//pubmed/25918418).

- **Study description**: Using 22 transcriptomes of developing maize leaves from dry seeds to 192 h post imbibition, 
we studied gene up- and down-regulation and functional transition during leaf development and inferred sets of strongly coexpressed genes. 
- **SRA Study ID**: [SRP044069](https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP044069)
- **Data Size**: 0.5 Tbp; and 0.3Tb in size
- Zea mays subsp. mays strain: White Crystal


